tasks/bes-reset.yml

```diff
@@ -1,2 +1,18 @@
 ---
-# TBD
\ No newline at end of file
+- name: Removing package
+  ansible.builtin.include_tasks: bes-remove.yml
```

Comment:
@asa1997 @Pranshu021 should be compliant with oah spec , bes-remove.yml not part of the spec